lib/open_project/global_roles/patches/role_patch.rb

```diff
@@ -25,7 +25,9 @@ def self.included(base)
 
 
       base.class_eval do
-        scope :givable, where(:builtin => 0, :type => 'Role').order('position')
+        scope :givable, -> {
+          where(builtin: 0, type: 'Role').order('position')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.